### PR TITLE
External provider: out of desktop app during login

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost/pull/24140

### CODE REVIEW

**Comment by cwarnermm** [comment]
@devinbinnie - I used a generic approach with this update, rather than explicitly calling out external providers to reach the widest audience possible. Thoughts?

File: source/collaborate/access-your-workspace.rst [hunk 1]
```diff
@@ -43,6 +43,9 @@ Mattermost is accessible through a URL link. You'll receive a link from your Mat
     3. Enter your user credentials to log into Mattermost. 
     4. The team that displays first in the team sidebar opens. If you're not a member of a team yet, you're prompted to select a team to join.
 
+    .. note::
+
+      Depending on how your user credentials are set up in Mattermost, you may temporarily leave the desktop app during login while authenticating your credentials. Once you're successfully logged in to Mattermost, you'll be returned to the desktop app.
```
Comment (devinbinnie) [inline source/collaborate/access-your-workspace.rst:48]: I'd specifically mention external providers here and maybe cite a couple examples (like Google, Office 365). We can be more specific and say OAuth and SAML providers, but that might not be helpful for people.

**Review by devinbinnie — CHANGES_REQUESTED** [review]

File: source/collaborate/access-your-workspace.rst (hunk 1 above)
Comment (cwarnermm) [reply source/collaborate/access-your-workspace.rst]: Thanks, @devinbinnie! Focusing the examples on well-known providers is a good customer-centric call. My latest commit includes your feedback. This PR is ready for your final review.

**Review by devinbinnie — APPROVED** [review]
Thanks Carrie!

**Pull request merged**